iceprod/server/data/www/submit.js

```diff
@@ -112,16 +112,14 @@ var Submission = (function( $ ) {
         },
         submit : function(num_jobs, gridspec, description) {
             private_methods.clean_json();
-            RPCclient('submit_dataset',{passkey:data.passkey,config:data.submit_data,njobs:num_jobs,gridspec:gridspec,description:description},callback=function(return_data){
-                $('#error').html('success');
-                window.location = '/dataset/' + return_data;
-            });
+            fetch_json('/datasets/', data.passkey, data.dataset, post=true);
+            fetch_json('/config/' + data.dataset.dataset_id, data.passkey, data.submit_data);
+            window.location = '/dataset/' + data.dataset.dataset_id;

```

Comment:
How is `data.dataset.dataset_id` getting set?  It should be in the response object for POST `/datasets`.

Reply:
8ccb242